sdk/tools/gen_baseaddress.py

```diff
@@ -356,6 +364,8 @@ def get_target_file(ntdll_path):
             return 'baseaddress_msvc.cmake'
         elif count > 3:
             return 'baseaddress_dwarf.cmake'
+        elif is_gcc() and is_x64():
```

Comment:
@ChaseKnowlden Do you think the very first `b'.rossym' in names` case could be modified as well? Because that was some sort of behaviour-based-on-file-contents (presence or absence of our in-house "rossym" debug symbols in the binaries) instead of a compiler choice.